docs/install/collecting-metrics/collector.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+  namespace: 
+data:
+  collector.yaml: |
+    receivers:
+      opencensus:
+        endpoint: "0.0.0.0:55678"
+
+    exporters:
+      logging:
+      prometheus:
+        endpoint: "0.0.0.0:8889"
+
+    service:
+      extensions:
+        health_check:
+        pprof:
+        zpages:
+      pipelines:
+        metrics:
+          receivers: [opencensus]
+          processors: []
+          exporters: [prometheus]
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otel-collector
+  namespace: metrics
+  labels:
+    app: otel-collector
+spec:
+  selector:
+    matchLabels:
+      app: otel-collector
+  replicas: 1  # This can be increased for a larger system.
+  template:
+    metadata:
+      labels:
+        app: otel-collector
+    spec:
+      containers:
+      - name: collector
+        args:
+        - --config=/conf/collector.yaml
+        image: otel/opentelemetry-collector:latest
+        resources:
+          requests:  # Note: these are suitable for a small instance, but may need to be increased for a large instance.
+            memory: 100Mi
+            cpu: 50m
+        ports:
+        - name: otel
+          containerPort: 55678
+        - name: prom-export
+          containerPort: 8889
+        - name: zpages  # A /debug page
+          containerPort: 55679
```

Comment:
I would argue that the internal metrics are useful too, particularly during initial setup.

Maybe announce the 8888 port here?

Reply:
The metrics describing the operation of the OpenTelemetry collector?

I didn't want to confuse things too much -- this was intended to be an example rather than a complete solution(for example, there's no dashboard, and I'm personally not interested in trying to maintain something like that dashboard).

Reply:
I see.  This works for me, then.  I see the value in keeping it simpler.